### PR TITLE
Updated HHAnalysis for phase2 analysis

### PR DESCRIPTION
**Pull request by mdelcourt**
Removing many producers
Added HGCAL electrons and updated working points

### CODE REVIEW

File: test/HHConfiguration.py [hunk 1]
```diff
@@ -58,11 +58,11 @@
             jetPtCut = cms.untracked.double(20),
 
             # BTAG INFO
-            # Working points from https://twiki.cern.ch/twiki/bin/viewauth/CMS/BtagRecommendation80XReReco
+            # Working points from https://twiki.cern.ch/twiki/bin/view/CMS/Phase2HGCRecipes#B_tagging
             discr_name =  cms.untracked.string("pfCombinedMVAV2BJetTags"),
```
Comment (OlivierBondu) [inline test/HHConfiguration.py:62]: don't you want to move to deepCSV ?

**Review by OlivierBondu — APPROVED** [review]
Thanks ! only one minor comment on my end

**Comment by OlivierBondu** [comment]
test

File: test/HHConfiguration.py (hunk 1 above)
Comment (mdelcourt) [reply test/HHConfiguration.py:62]: Well, I don't know... Are the b-jets rather central ?
> Note that for |eta|<1.5, cMVAv2 should be used, while for |eta|>1.5, DeepCSV is recommended.

Comment (OlivierBondu) [reply test/HHConfiguration.py:62]: it should yes, but let's see on your plots :smile:

**Comment by OlivierBondu** [comment]
thanks!

**Pull request merged**